frontend/src/Channels.elm

```diff
@@ -0,0 +1,17 @@
+module Channels exposing (channelsJSON)
+
+
+channelsJSON : String
+channelsJSON =
+    """
+[
+    {
+        "channel": "21.11",
+        "status": "stable"
+    },
+    {
+        "channel": "unstable",
+        "status": "rolling"
+    }
+]
+"""
```

Comment:
It would be better to make this a JSON file and embed it in the Elm code somehow, if we want to be able to update it easily from outside.

Reply:
For example using https://github.com/emmabastas/elm-embed

Reply:
This probably adds more complexity than it solves though

Reply:
Can we do this in the next step? Because we need to modify the deployment (see https://github.com/emmabastas/elm-embed#linux) and it might not be as trivial